app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -153,13 +153,14 @@ class WireActivityViewModel @Inject constructor(
     }
 
     private fun observeUpdateAppState() {
-        viewModelScope.launch(dispatchers.io()) {
-            observeIfAppUpdateRequired(BuildConfig.VERSION_CODE)
-                .distinctUntilChanged()
-                .collect {
-                    globalAppState = globalAppState.copy(updateAppDialog = it)
-                }
-        }
+        appLogger.d("bring me back")
+//        viewModelScope.launch(dispatchers.io()) {
```

Comment:
Is this a TODO: to be implemented in the next PR?

Reply:
yes